Simplified/AxisService/NYPLAxisDRMAuthorizer.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  NYPLAxisDRMAuthorizer.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-13.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+#if AXIS
+@objcMembers
+class NYPLAxisDRMAuthorizer: NSObject, NYPLDRMAuthorizing {
+  
+  static let deviceIDKey = "NYPLAxisDRMAuthorizerKey"
+  
+  static let sharedInstance = NYPLAxisDRMAuthorizer()
+  
+  static var deviceID: String {
+    if
+      let id = UserDefaults.standard.string(forKey: NYPLAxisDRMAuthorizer.deviceIDKey) {
+      return id
+    }
+    
+    let id = UUID().uuidString
+    UserDefaults.standard.set(id, forKey: NYPLAxisDRMAuthorizer.deviceIDKey)
+    return id
+  }
+  
+  /// Not applicable for Axis
+  var workflowsInProgress: Bool {
+    return false
+  }
+  
+  /// We're always returning true since Axis does not deauthorize a user.
+  func isUserAuthorized(_ userID: String!,
+                        withDevice device: String!) -> Bool {
+   return true
+  }
+  
+  /// As of now, we're creating a deviceID which stays saved until the user uninstalls the app.
+  func authorize(withVendorID vendorID: String!,
+                 username: String!,
+                 password: String!,
+                 completion: ((Bool, Error?, String?, String?) -> Void)!) {
+    
+    completion(true, nil, NYPLAxisDRMAuthorizer.deviceID, username)
```

Comment:
Since the NYPLDRMAuthorizing object does not do anything here, do you think we're violating( or not properly following) the Liskov Substitution Principal here @ettore @ErnestFan?

Reply:
ok I confess I had to look that up 😅 but I don't think so we're violating it... considering NYPLAxisDRMAuthorizer is a subtype of NYPLDRMAuthorizing, and the latter is a protocol with no implementation, all we can ask is, can we use the concrete class anywhere we need the protocol? The answer is definitely yes.

Also, it's not entirely true this authorizer does nothing. It creates a device ID, persists it, and returns it consistently. The fact that there's no authorization process seems ok to me. A detail if you will.... basically Axis is saying, if you can use the DRM stuff on device X, you can also use it on device Y, whatever that is.